### PR TITLE
Keep priority matadata producer factory consistent with predicates factory

### PR DESCRIPTION
**Pull request by guangxuli**
**What this PR does / why we need it**:

Keep the priority metadata producer factory have the same name style with predicates producer.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
None
**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

File: plugin/pkg/scheduler/factory/plugins.go [hunk 1]
```diff
@@ -46,9 +46,8 @@ type PluginFactoryArgs struct {
 	HardPodAffinitySymmetricWeight int32
 }
 
-// MetadataProducerFactory produces MetadataProducer from the given args.
-// TODO: Rename this to PriorityMetadataProducerFactory.
-type MetadataProducerFactory func(PluginFactoryArgs) algorithm.MetadataProducer
+// PriorityMetadataProducerFactory produces PriorityMetadataProducer from the given args.
+type PriorityMetadataProducerFactory func(PluginFactoryArgs) algorithm.MetadataProducer
```
Comment (bsalamat) [inline plugin/pkg/scheduler/factory/plugins.go]: We should rename this only after we have a specific type for PriorityMetadata. We don't have that today. That's the first step before renaming this. I am specifically referring to this:
https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/scheduler/algorithm/types.go#L49

**Review by bsalamat — CHANGES_REQUESTED** [review]

File: plugin/pkg/scheduler/factory/plugins.go (hunk 1 above)
Comment (guangxuli) [reply plugin/pkg/scheduler/factory/plugins.go]: oh, got it, make a misunderstanding, i should hold this PR. I think we could use this after all priority functions change to map/reduce pattern.

**Comment by bsalamat** [comment]
@guangxuli I guess you can close this PR

**Comment by guangxuli** [comment]
oh, sure, i will close this.

**Pull request closed without merging**